server/.flaskenv

```diff
@@ -0,0 +1 @@
+
```

Comment:
Does flask recognize this file by default to load the env?

We could also use a common package python-dotenv as that's also a very common standard too

Reply:
yes, flask recognizes this file

Reply:
Can you research a .flaskenv vs .env and it's use cases when it comes to flask? We can decide on a unified environment variable file or separate them as well

Reply:
Alright cool let's keep both of them in our ignore then so we could use them both